packages/http/src/validator/deserializers/index.ts

```diff
@@ -1,29 +1,17 @@
-import { HttpParamStyles } from '@stoplight/types';
-
-import { HttpParamDeserializerRegistry } from './registry';
-import {
-  DeepObjectStyleDeserializer,
-  DelimitedStyleDeserializer,
-  FormStyleDeserializer,
-  SimpleStyleDeserializer,
-} from './style';
-import { LabelStyleDeserializer } from './style/label';
-import { MatrixStyleDeserializer } from './style/matrix';
-
-export const header = new HttpParamDeserializerRegistry([new SimpleStyleDeserializer()]);
-
-export const query = new HttpParamDeserializerRegistry([
-  new FormStyleDeserializer(),
-  new DelimitedStyleDeserializer('%20', HttpParamStyles.SpaceDelimited),
-  new DelimitedStyleDeserializer('|', HttpParamStyles.PipeDelimited),
-  new DelimitedStyleDeserializer(',', HttpParamStyles.CommaDelimited),
-  new DeepObjectStyleDeserializer(),
-]);
-
-export const path = new HttpParamDeserializerRegistry([
-  new SimpleStyleDeserializer(),
-  new LabelStyleDeserializer(),
-  new MatrixStyleDeserializer(),
-]);
+import { deserializeSimple as simple } from './style/simple';
+import { deserializeForm as form } from './style/form';
+import { deserializeDeepObject as deepObject } from './style/deepObject';
+import { deserializeLabel as label } from './style/label';
+import { deserializeMatrixStyle as matrix } from './style/matrix';
```

Comment:
Lines 1-4 are not postfixed with "Style" while "Matrix" is, is it intended?

Reply:
It was not. Fixed in 8e13d24